otterdog/jetty.jsonnet

```diff
@@ -298,7 +298,13 @@ orgs.newOrg('jetty') {
       dependabot_security_updates_enabled: true,
       description: "Eclipse Jetty® - Test Helper",
       has_wiki: false,
-      homepage: "https://eclipse.dev/jetty",
+      homepage: "https://jetty.org/",
```

Comment:
all the other repos have https://eclipse.dev/jetty as homepage. Not sure if this difference is by accident or intentional.

Reply:
Eh, not really an accident.
But it really doesn't matter, as the homepage is in transition, either URL is valid right now.